akka-persistence-typed/src/main/scala/akka/persistence/typed/SnapshotAdapter.scala

```diff
@@ -7,7 +7,7 @@ package akka.persistence.typed
 /**
  * Facility to convert snapshots from and to a specialized data model.
  * Can be used when migration from different state types e.g. when migration
- * from Persistent FSM to Typed Persistence.
+ * from Persistent FSM to `akka.persistence.typed.Persistence`.
```

Comment:
Make it a link even with `[[akka.persistence.typed.Persistence]]`

Reply:
akka.persistence.typed.Persistence isn't a class

I think we can write: Typed `EventSourcedBehavior` to avoid double references to javadsl and scaladsl